### DISCUSSION TITLE
Umzug auf neuen Server (Verlaufsdaten)

### DISCUSSION THREAD

**Discussion by nicodonner** [solectrus/solectrus#2694] [Q&A] [answered]
Hallo zusammen,

ich hoffe hier bin ich richtig und bekomme evtl. Hilfe. 

Da mittlerweile mein Raspberry PI 4 an seine Grenzen kommt bin ich aktuell am Umzug auf ein Intel Nuc mit Proxmox. 

Der LXC Container läuft bereits, Docker und Solectrus laufen soweit - Einstellungen sind alle gemacht. 

Ich weiß aktuell nur noch nicht wie ich die alten Daten von meinem PI auf den LXC Container bekomme, damit ich wieder auf dem "alten Stand" bin - mittlerweile 1 Jahr gesammelte Daten vom Solectrus würde ich ungern verschenken wollen. 

Kann mir jemand einen Tipp geben? 

Viele Dank 
Gruß

**Comment by ledermann** [comment, marked as answer]
Ja, du bist hier richtig :)

Ein Umzug der Daten von SOLECTRUS ist meiner Erfahrung nach möglich, indem einfach die Docker-Volumes kopiert werden. Also: Auf altem System alle Docker-Container regulär beenden (`docker-compose down`), insbesondere InfluxDB, PostgreSQL und Redis. Dann die drei zugehörigen Ordner auf das neue System kopieren (mit allen Unterordnern und Dateien) und dort die Container neu starten (`docker-compose up`).

Professioneller wäre sicherlich, die zugehörigen Backup/Restore-Tools zu verwenden, aber das erfordert etwas Einarbeitung. Ein einfaches Kopieren der Ordner geht auch, wenn die Docker-Container zum Kopierzeitpunkt beendet sind.

Bezüglich "Raspi an seinen Grenzen": Die aktuelle Version `v0.14.1` beansprucht deutlich mehr Speicherplatz, das war eigentlich nicht beabsichtigt. Mittels env `WEB_CONCURRENCY=0" kann der Speicherverbrauch wieder auf den früheren Stand zurückgedreht werden. Vielleicht rettet das deinen Raspi...

**Reply by nicodonner** [reply]
Sorry für die späte Rückmeldung. 

Ich hatte ein wenig Ärger mit dem Zugriff auf den Ordner PostgreSQL - was aber auch gelöst werden konnte. 

Vielen Dank für deine Hilfe und generell für dieses super Projekt!

**Reply by ledermann** [reply]
Prima, besten Dank für die Rückmeldung!
